lib/CMakeLists.txt

```diff
@@ -413,13 +413,11 @@ endif()
 
 if(QUDA_QMP)
   if(QUDA_DOWNLOAD_USQCD AND NOT QMP_FOUND)
-    add_dependencies(quda QMP)
-    add_dependencies(quda_cpp QMP)
+    add_dependencies(quda QMPBuild)
```

Comment:
QMP is now the namespace for the imported QMP targets (QMP::qmp). So I could not leave these as 'QMP' but needed to rename them. We can use a different name if desired.